layouts/partials/footer.html

```diff
@@ -36,6 +38,14 @@
             <li><a href="https://filecoin.io/">About</a></li>
             <li><a href="https://filecoin.io/faqs/">FAQ</a></li>
           </ul>
+          <div class="stacked">
+            <ul class="sitemap unstyled">
+              <li class="sitemap-head">Other Projects</li>
+              <li><a href="http://libp2p.io/">libp2p</a></li>
+              <li><a href="http://ipld.io/">IPLD</a></li>
+              <li><a href="http://multiformats.io/">Multiformats</a></li>
```

Comment:
We should use `https` as for these sites. Ideally DNS redirects should exist for all of these by default but that isn't the case at the moment.

Reply:
Apologies, fixed.